templates/terraform/custom_flatten/get_pwd_from_config.erb

```diff
@@ -0,0 +1,3 @@
+func flatten<%= prefix -%><%= titlelize_property(property) -%>(v interface{}, d *schema.ResourceData) interface{} {
```

Comment:
can the name of this file be something more specific, since the structure of these fields is specific to this resource?